src/components/animated-menu-button/animated-menu-button.js

```diff
@@ -218,6 +218,7 @@ class AnimatedMenuButton extends React.Component {
         data-element='close'
         key='close'
         onClick={ this.closeHandler }
+        onKeyPress={ this.closeHandler }
```

Comment:
is this any key? do you mean this to be on enter or something?

Reply:
I've updated this so it only responds to the enter key.